### PR TITLE
fixed imaged for safari and updated text width

### PR DESCRIPTION
**Pull request by sei1122**
Update image size to work on Safari. set the text width to match to the Typography page.

### CODE REVIEW

File: UXWLogosAndIcons.js [hunk 1]
```diff
@@ -94,12 +94,12 @@ function UXWLogosAndIcons(props) {
...
                   <div className="py-2 pr-4">
                     <a href="assets/library/icongraphy.zip" download>
-                      <i className="bi bi-download"></i>
+                      <i className="bi bi-download text-xl"></i>
                     </a>
                   </div>
                 </div>
                 <div className="flex flex-row justify-center bg-gray-100 rounded-b-lg">
-                  <img className="w-3/5 py-12" src="assets/icongraphy.png" alt="" />
+                  <img className="w-3/5 py-12 icongraphy" src="assets/icongraphy.png" alt="" />
```
Comment (uxwizards) [inline UXWLogosAndIcons.js:102]: Use `my-12` instead of `py-12` here

**Comment by uxwizards** [comment]
Once you merge this in, are you closing `seiko7`?

**Comment by sei1122** [comment]
> Once you merge this in, are you closing `seiko7`?
I'm now branching seiko8. I don't know what to do seiko7 now. 
Iconography image on Safari was still wired so I updated again

**Comment by uxwizards** [comment]
Looks stretched for me on Firefox. Not sure if someone else is seeing that.

**Pull request closed without merging**